src/lambda-invoker.ts

```diff
@@ -32,7 +31,7 @@ export const invokeLambda = async (params: { payload: string, functionName: stri
     throw new Error(res.Payload?.toString())
   }
 
-  const response = safeParseJson(res.Payload?.toString())
```

Comment:
@iDanielBot the SDK v3 supports more response type, hence the `toString()` is not enough anymore. See this discussion for further information: https://github.com/aws/aws-sdk-js-v3/issues/2252